<?xml version="1.0"?>
<launch>
  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="ImagePublisherNodelet" args="load tuw_image_publisher/tuw_image_publisher_nodelet standalone_nodelet" output="screen">
  </node>                 
</launch>